### PR TITLE
feat: [PIE-3300]: Cypress tests for Service/Environment edit flow

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Summary:

Added cypress test cases for service/environment edit flow

##### Jira Links:

https://harness.atlassian.net/browse/PIE-3300

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

File: cypress/fixtures/ng/api/servicesV2/servicesUpdate.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "status": "SUCCESS",
+  "data": {
+    "service": {
+      "accountId": "px7xd_BFRCi-pfWPYXVjvw",
+      "identifier": "testService",
+      "orgIdentifier": "harness",
+      "projectIdentifier": "Chirag",
```
Comment (PratyushGar) [inline cypress/fixtures/ng/api/servicesV2/servicesUpdate.json:8]: Please ensure same projectIdentifier in all mocks.

File: cypress/fixtures/ng/api/servicesV2/servicesUpdate.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "name": "NewtestService",
+      "description": "",
+      "deleted": false,
+      "tags": {},
+      "yaml": "service:\n  name: \"NewtestService\"\n  identifier: \"NewS\"\n  orgIdentifier: \"harness\"\n  projectIdentifier: \"Chirag\"\n  description: \"\"\n  tags: {}\n"
+    },
```
Comment (PratyushGar) [inline cypress/fixtures/ng/api/servicesV2/servicesUpdate.json:14]: same here.

File: cypress/integration/70-pipeline/Environments.spec.ts [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import { environmentRoute, environmentsCall } from '../../support/70-pipeline/constants'
 
 describe('Environment for Pipeline', () => {
+  const envUpsertCall = '/ng/api/environmentsV2/upsert?routingId=accountId&accountIdentifier=accountId'
+  const envUpdateList =
+    '/ng/api/environmentsV2?routingId=accountId&accountIdentifier=accountId&orgIdentifier=default&projectIdentifier=project1&page=0&size=10'
```
Comment (PratyushGar) [inline cypress/integration/70-pipeline/Environments.spec.ts:6]: apiRoutes and call declarations can be moved to 70-pipeline/constants.

**Review by PratyushGar — CHANGES_REQUESTED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Comment by Sagar-Rajput7** [comment]
retrigger typecheck

**Review by PratyushGar — APPROVED** [review]

File: cypress/integration/70-pipeline/Environments.spec.ts (hunk 3 above)
Comment (Sagar-Rajput7) [reply cypress/integration/70-pipeline/Environments.spec.ts]: Resolved

File: cypress/fixtures/ng/api/servicesV2/servicesUpdate.json (hunk 2 above)
Comment (Sagar-Rajput7) [reply cypress/fixtures/ng/api/servicesV2/servicesUpdate.json:14]: Resolved

File: cypress/fixtures/ng/api/servicesV2/servicesUpdate.json (hunk 1 above)
Comment (Sagar-Rajput7) [reply cypress/fixtures/ng/api/servicesV2/servicesUpdate.json]: Resolved

File: cypress/integration/70-pipeline/Services.spec.ts [hunk 4]
```diff
@@ -1,6 +1,9 @@
 import { servicesRoute, servicesCall } from '../../support/70-pipeline/constants'
 
 describe('Services for Pipeline', () => {
+  const servicesUpsertCall = '/ng/api/servicesV2/upsert?routingId=accountId&accountIdentifier=accountId'
+  const servicesUpdateList =
+    '/ng/api/servicesV2?routingId=accountId&accountIdentifier=accountId&orgIdentifier=default&projectIdentifier=project1&size=10&page=0'
```
[the remark this answers is not in the document]
Comment (Sagar-Rajput7) [reply cypress/integration/70-pipeline/Services.spec.ts]: Resolved

**Pull request merged**